src/System.Diagnostics.Tracing/src/ApiCompatBaseline.net462.txt

```diff
@@ -0,0 +1,7 @@
+# DevDiv: Bug 1179822: ProjectN is missing the implementation of some System.Diagnostics.Tracing contract API
```

Comment:
updated comment to desktop. I will be opening a bug for every baseline file i create. So once we have the targeting pack for net462 and this isn't needed we can remove the baseline.